client/question/services/FeedbackGeneratorService.js

```diff
@@ -257,40 +322,87 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     /**
+     * Concats two strings to create testCaseKey
```

Comment:
Nits:
- Concats --> Concatenates
- Add period at end.
- The main purpose of this function (from an outsider's perspective) is to create a key that uniquely identifies a test case. So I wouldn't worry about the implementation details; a better docstring would be "Generates a unique test case key for the given (test suite, test case) pair."

Reply:
Changed to suggested text.